### DISCUSSION TITLE
code style in util.py

### DISCUSSION THREAD

**Discussion by artyrian** [eternnoir/pyTelegramBotAPI#1757] [Ideas]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?
4.7.0

2. What OS are you using?
MacOS 12.5

3. What version of python are you using?
python 3.8

I'm reading the util.py (https://github.com/eternnoir/pyTelegramBotAPI/blob/master/telebot/util.py)
and noticed a difference in style

so I suggest a small code style fix for quotes for example
cause it's would be better for readability and consistency

Some link for recomendation: https://docs.ckan.org/en/ckan-2.1.5/python-coding-standards.html#use-single-quotes

**Comment by artyrian** [comment]
so there is a small fix for this.
https://pastebin.com/K93a8PJW

is a pull requests may be opened for everyone?

**Comment by artyrian** [comment]
also fix commit in a forked repo:

https://github.com/artyrian/pyTelegramBotAPI/commit/e03cdda4e0dfac5fdfb4781eb5070b4951b23856

and created pr: https://github.com/eternnoir/pyTelegramBotAPI/pull/1756

**Comment by Badiboy** [comment]
Not an issue.

**Comment by Badiboy** [comment]
_No need to tune code style and waste the time.
If you want to help - better make something really useful._
